spec/item_spec.rb

```diff
@@ -0,0 +1,70 @@
+require './item'
+require_relative '../models/genre'
+require './author'
+require './label'
+
+describe Item do
+  item = Item.new('2010/01/01')
+  let(:genre) { Genre.new('Horror') }
+  let(:author) { Author.new('Min', 'Lu') }
+  let(:label) { Label.new('L1') }
+  it 'should display the publish date' do
+    expect(item.publish_date).to eq('2010/01/01')
+  end
+
+  describe '#time_difference' do
+    it 'returns the correct time difference' do
+      item = Item.new('2000-01-01')
+      expected_time_difference = Date.today.year - Date.parse('2000-01-01').year
+      expect(item.time_difference).to eq(expected_time_difference)
+    end
+  end
+
+  describe '#can_be_archived?' do
+    it 'returns true if the item can be archived' do
+      item = Item.new("#{Date.today.year - 11}-01-01")
+      expect(item.can_be_archived?).to be true
+    end
+    it 'returns false if the item cannot be archived' do
+      item = Item.new("#{Date.today.year - 9}-01-01")
+      expect(item.can_be_archived?).to be_falsey
+    end
+  end
+
+  describe '#move_to_archive' do
+    it 'return true if the item can be archived' do
+      item = Item.new("#{Date.today.year - 11}-01-01")
+      item.move_to_archive
+      expect(item.move_to_archive).to be true
+    end
+
+    it 'return false if the item cannot be archived' do
+      item = Item.new("#{Date.today.year - 9}-01-01")
+      expect(item.move_to_archive).to be false
+    end
+  end
```

Comment:
- [ ] Kindly make sure that all tests added are passing, this test  is failng 👍🏿 

![Screenshot_207](https://github.com/MayPyone/Catalog-of-my-things/assets/69638013/1c79503d-8c8f-40d0-9bbf-903bdcd4a06e)